### ISSUE TITLE
Frigate lovelace card: Loading first video from media folder

### ISSUE THREAD

**Issue by woempiej** [blakeblackshear/frigate#9827]
### Describe the problem you are having

Hi,

I'm using the Frigate Lovelace card. Since the latest update the Lovelace card doesn't show the 1st (latest) video anymore.
I'm using the card in the "most recent clip" view.

![afbeelding](https://github.com/blakeblackshear/frigate/assets/62903344/cab9fddb-d85e-45af-9e85-963ff21ca6e6)

I need to
Android HA app:
Swipe to the previous video and then swipe back to the newest video to be able to watch the newest video.
Firefox browser:
Navigate to another dashboard tab and go back to my "security" tab to be able to watch the newest video.
Chrome browser:
Navigate to another dashboard tab and go back to my "security" tab to be able to watch the newest video.

I did think about the browser cache and history wipe.

No error(s) logs in the HA log
No error(s) in the Frigate log

Yesterday I rebuild all of the Frigate installation:
Deleted HA add-on
Deleted HA integration > Wich the Lovelace card is comming from
Deleted all Frigate related files via the file editor
Deleted all Media
Media > Frigate (Integration folder)
Media > My Media > frigate (Add-On folder)

Deleted the Lovelace card and builded it back up via the GUI

### Version

0.13.1

### Frigate config file

```yaml
mqtt:
  host: 192.168.X.X
  user: mqtt-username
  password: password

detectors:
  coral:
    type: edgetpu
    device: usb

birdseye:
  enabled: False

go2rtc:
  streams:
    frigate_cam1:
      - rtsp://camera1
    frigate_cam2:
      - rtsp://camera2
    frigate_cam3:
      - rtsp://camera3

################################################################################
cameras:
  frigate_cam1:
    ffmpeg:
      inputs:
        - path: rtsp://cam1/videoSub
          roles:
            - detect
        - path: rtsp://cam1/videoMain
          roles:
            - record
    detect:
      width: 1280
      height: 720
    zones:
      zone_0_cam1:
        coordinates: 309,720,1280,720,1280,355,205,96
    objects:
      track:
        - person
      filters:
        person:
          min_area: 5000
          mask: 1280,0,1280,164,191,83,302,720,0,720,0,0
    motion:
      mask:
        - 0,39,374,38,373,0,0,0
        - 1280,0,1280,164,191,83,302,720,0,720,0,0
    record:
      enabled: True
      events:
        objects:
          - person
        required_zones:
          - zone_0_cam1
        retain:
          default: 5
          mode: active_objects
    snapshots:
      enabled: True
      clean_copy: False
      retain:
        default: 5
################################################################################
  frigate_cam2:
    ffmpeg:
      inputs:
        - path: rtsp://cam2/videoSub
          roles:
            - detect
        - path: rtsp://cam2/videoMain
          roles:
            - record
    detect:
      width: 1280
      height: 720
    zones:
      zone_0_cam2:
        coordinates: 484,172,740,25,846,720,137,720
    objects:
      track:
        - person
      filters:
        person:
          min_area: 3000
          mask: 0,0,733,0,686,159,455,173,285,506,311,720,0,720,0,346
    motion:
      mask: 0,39,374,38,373,0,0,0
    record:
      enabled: True
      events:
        objects:
          - person
        required_zones:
          - zone_0_cam2
        retain:
          default: 5
          mode: active_objects
    snapshots:
      enabled: True
      clean_copy: False
      retain:
        default: 5
################################################################################
  frigate_cam3:
    ffmpeg:
      inputs:
        - path: rtsp://cam3/videoSub
          roles:
            - detect
        - path: rtsp://cam3/videoMain
          roles:
            - record
    detect:
      width: 1280
      height: 720
    zones:
      zone_0_cam3:
        coordinates: 0,720,1280,720,1107,89,0,82
    objects:
      track:
        - person
      filters:
        person:
          min_area: 5000
          mask:
            - 769,93,845,92,848,194,767,194
            - 0,421,142,391,256,392,413,346,410,243,366,213,282,194,146,195,0,215
    motion:
      mask:
        - 0,96,739,86,741,184,856,181,853,86,1130,96,1114,235,1280,471,1280,105,1280,0,0,0
        - 0,421,142,391,256,392,413,346,410,243,366,213,282,194,146,195,0,215
    record:
      enabled: True
      events:
        objects:
          - person
        required_zones:
          - zone_0_cam3
        retain:
          default: 5
          mode: active_objects
    snapshots:
      enabled: True
      clean_copy: False
      retain:
        default: 5
```

### Relevant log output

```shell
No relevant logs
```

### FFprobe output from your camera

```shell
[{"return_code":0,"stderr":"","stdout":{"programs":[],"streams":[{"avg_frame_rate":"31/6","codec_long_name":"H.264/AVC/MPEG-4AVC/MPEG-4part10","display_aspect_ratio":"16:9","height":720,"width":1280},{"avg_frame_rate":"0/0","bit_rate":"64000","codec_long_name":"PCMmu-law/G.711mu-law"}]}},{"return_code":0,"stderr":"","stdout":{"programs":[],"streams":[{"avg_frame_rate":"31/2","codec_long_name":"H.264/AVC/MPEG-4AVC/MPEG-4part10","display_aspect_ratio":"16:9","height":720,"width":1280},{"avg_frame_rate":"0/0","bit_rate":"64000","codec_long_name":"PCMmu-law/G.711mu-law"}]}}]
```

### Operating system

HassOS

### Install method

HassOS Addon

### Coral version

USB

### Network connection

Wired

### Camera make and model

Foscam FI9900P

### Any other information that may be helpful

Lovelace Card config
```yaml
  - type: custom:frigate-card
    cameras:
      - camera_entity: camera1
      - camera_entity: camera2
      - camera_entity: camera3
    view:
      default: clip
      update_force: true
      scan:
        show_trigger_status: true
    menu:
      style: outside
      buttons:
        media_player:
          enabled: false
        timeline:
          enabled: false
        live:
          enabled: false
        snapshots:
          enabled: false
    media_viewer:
      controls:
        title:
          mode: none
        thumbnails:
          mode: none
        timeline:
          mode: below
          window_seconds: 28800
    media_gallery:
      controls:
        thumbnails:
          show_favorite_control: false
          show_timeline_control: false
          show_download_control: false
        filter:
          mode: none
```

**Comment by NickM-27** [comment]
this is the wrong repo, see https://github.com/dermotduffy/frigate-hass-card

**Comment by woempiej** [comment]
Hi,

Turns out this issue is not Lovelace card related, can you re-open this issue?

**Comment by kenni** [comment]
If anyone else experience the issue and ends up at this bug report: It isn't related to Frigate, it is apparently a bump in the HLS player source code included in HA v2024.2. Downgrading to HA v2024.1.6 reverts the issue. See upstream bug report here (no fix as of today):
https://github.com/home-assistant/frontend/issues/19799
